lib/plugin.js

```diff
@@ -1,20 +1,38 @@
+<% if (options.methods) { %>
+import { <%= options.methods.join(', ') %> } from 'date-fns'
+<% } else { %>
 import * as dateFns from 'date-fns'
+<% } %>
 
 <% if (options.locales) { %>
 import { <%= options.locales.join(', ') %> } from 'date-fns/locale'
 <% } %>
 
 export default (ctx, inject) => {
+  <% if (options.methods) { %>
+  const dateFns = { <%= options.methods.join(', ') %> }
+  <% } %>
   const dateObj = {...dateFns}
 
-  dateObj.format = (date, format, options) => {
-    const parsedDate = typeof date !== 'string' ? date : dateFns.parseISO(date)
 
-    return dateFns.format(parsedDate, format || '<%= options.format %>', mergeOptions(options))
+  function parsedDate(date) {
+    if(dateFns.parseISO) {
```

Comment:
Add space before open parenthesis

Reply:
not sure what you mean as there is a space before the {

Reply:
Its here `if (`

Reply:
ah okay